.ci/msys_test.bat

```diff
@@ -0,0 +1,7 @@
+:: FIXME(equalsraf) currently not all tests work on Windows
+:: for now just run what we can
+mingw32-make functionaltest TEST_FILE=test\functional\eval\printf_spec.lua VERBOSE=1 || goto :error
```

Comment:
Ah the other culprit seems to be luajit crashes when running certain test groups (cannot pinpoint individual test), assert in lj_gc.c Line 666 IIIRC.